services/src/main/java/org/fao/geonet/api/thesaurus/KeywordsApi.java

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
Would be better located in package org.fao.geonet.api.registries.vocabularies;
I'll take care of the move.